### ISSUE TITLE
[CLOSED] Python Interface on Windows

### ISSUE THREAD

**Issue by eriq-augustine** [linqs/psl#250]
<a href="https://github.com/eriq-augustine"><img src="https://avatars0.githubusercontent.com/u/337857?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [eriq-augustine](https://github.com/eriq-augustine)**
_Thursday Apr 11, 2019 at 15:10 GMT_
_Originally opened as https://github.com/eriq-augustine/psl/issues/203_

----

There seems to be some issues with the Python interface on Windows.

Reports are that it stems from differences in the asyncio implementation on Windows.

**Comment by eriq-augustine** [comment]
<a href="https://github.com/eriq-augustine"><img src="https://avatars0.githubusercontent.com/u/337857?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [eriq-augustine](https://github.com/eriq-augustine)**
_Thursday Apr 11, 2019 at 15:14 GMT_

----

Reports of this error:
```
 File "C:\python-3.7\lib\asyncio\base_events.py", line 463, in _make_subprocess_transport
    raise NotImplementedError
```

**Comment by eriq-augustine** [comment]
<a href="https://github.com/eriq-augustine"><img src="https://avatars0.githubusercontent.com/u/337857?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [eriq-augustine](https://github.com/eriq-augustine)**
_Sunday May 12, 2019 at 07:50 GMT_

----

I think this should be fixed since we moved away from asyncio in cdcf4cd8776ff21121a1a5111be21d158fb20e22.

We don't have any Windows CI, but we will close this for now as we stuck to only platform-safe parts of subprocess.
